### PR TITLE
DA-363: Export Query results as SQL++ inserts/upserts

### PR DESCRIPTION
**Pull request by teetangh**

### CODE REVIEW

**Review by deniswsrosa — CHANGES_REQUESTED** [review]
- revert the gradle.xml file
-  add a divider above the "SQL++ UPSERT" option
- Could you please open a popup asking the user for the collection name instead of just adding "my_collection"? The phrase could be something like "Inform the name of the collection that should be used in the dataset" . This way the user doesn't need to open the dataset and replaced "my_collection" with that he/she wants.

Everything else looks fine for me.

File: .idea/gradle.xml
[the remark this answers is not in the document]
Comment (teetangh) [reply .idea/gradle.xml:1]: Have reverted the file. Already in gitignore as .idea/gradle.xml

**Comment by deniswsrosa** [comment]
@teetangh 2 small fixes:

- You need to validate that the user has specified a name for the collection
- You need to check if there are any invalid characters in the collection's name. Note that the name could be fully qualified (`travel-sample`.inventory.some_collection)

**Review by deniswsrosa — CHANGES_REQUESTED** [review]
Check my comment in the PR. The collection name needs to be validated.

**Review by deniswsrosa — APPROVED** [review]

**Pull request merged**